types/start-server-webpack-plugin/index.d.ts

```diff
@@ -0,0 +1,53 @@
+// Type definitions for start-server-webpack-plugin 2.2
+// Project: https://github.com/ericclemmons/start-server-webpack-plugin
+// Definitions by: AGenson <https://github.com/AGenson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Typescript Version: 3.8
```

Comment:
This probably is not needed, no version specific feature is being used here I think.
Webpack types itself are also very loose on TS version.
```suggestion

```